nservicebus/messaging/send-a-message.md

```diff
@@ -1,27 +1,42 @@
 ---
-title: Sending a Message
-summary: Describes how to send a message
+title: Sending messages
+summary: Describes how to send messages
 tags: []
 redirects:
 - nservicebus/how-do-i-send-a-message
 ---
 
-Message sending involves using Send operation that takes an argument of a message instance to be delivered. The details differ slightly between version of NServiceBus, as shown in following snippets.
+NServiceBus allows you to send different types of messages (see [Messages, Events and Commands](messages-events-commands.md)) to any endpoint. Messages can be sent from outside or inside the message processing pipeline:
 
-Here's how a message is sent by directly using the instance of the endpoint
+## Outside the pipeline
+
+Messages are sent outside the pipeline when the send operation is not related to an incoming message. Examples may be:
```

Comment:
add an extra space before starting lists. some MD editors dont like the missing line